app/javascript/packs/components/Header.jsx

```diff
@@ -20,35 +20,62 @@ const Logo = () => (
 const useStyles = makeStyles((theme) => ({
   root: {
     backgroundColor: theme.palette.background.paper,
+    transform: "none !important",
     [theme.breakpoints.up("sm")]: {
       paddingLeft: "30px",
       paddingRight: "30px",
     },
   },
-  logo: {
-    [theme.breakpoints.down("sm")]: {
-      maxWidth: "30vw",
-      "& > svg": {
-        maxWidth: "100%",
-      },
+  [theme.breakpoints.down("sm")]: {
+    bar: {
+      flexDirection: "column",
+    },
+    logo: {
+      maxWidth: "60vw",
+    },
+    logoSvg: {
+      maxWidth: "100%",
+    },
+    sticky: {
+      marginTop: "90px",
+    },
+    stickyNav: {
+      position: "fixed",
+      boxShadow: "0 0 2px 2px rgba(0,0,0,0.1)",
+      top: "0",
+      zIndex: "100",
+      padding: "20px 15px",
+      background: "white",
+    },
+    hideLogo: {
+      display: "none",
     },
   },
 }));
 
 const Header = () => {
   const classes = useStyles();
+  const [isScrolled, setIsScrolled] = useState(false);
+  const headerDistance = -50;
+  useScrollPosition(({ currPos }) => {
+    setIsScrolled(currPos.y < headerDistance);
+  });
   return (
     <AppBar elevation={0} position="static" className={classes.root}>
       <Box
         p={2}
         display="flex"
         alignItems="center"
         justifyContent="space-between"
+        className={`${classes.bar} ${isScrolled ? classes.sticky : ""}`}
       >
-        <Link className={classes.logo} to="/">
-          <Logo />
+        <Link
+          className={`${classes.logo} ${isScrolled ? classes.hideLogo : ""}`}
+          to="/"
+        >
+          <Logo className={classes.logoSvg} />
         </Link>
-        <Nav />
+        <Nav className={`${isScrolled ? classes.stickyNav : ""}`} />
```

Comment:
Oh nice! I was thinking more like sending `isScrolled` as a prop to Nav and then defining the `stickyNav` class in Nav.jsx but this is good!